eng/scripts/Verify-And-Merge-PRs.ps1

```diff
@@ -9,6 +9,7 @@ param(
 $ReadyForMerge = $true
 $mergablePRs = @()
 $headers = @{ }
+$RetryCount = 3
```

Comment:
We can give this a try but we might consider more retries and perhaps a sleep between tries